.github/workflows/main.yml

```diff
@@ -69,7 +70,8 @@ jobs:
         run: docker tag prairielearn/plbase:${{ env.COMMIT_SHA }} prairielearn/plbase:latest
 
       - name: Build the prairielearn docker image
-        run: docker build -t prairielearn/prairielearn:${{ env.COMMIT_SHA }} .
+        run: |
```

Comment:
Does this need `type=registry` to match the above? Or is that somehow implied?

Reply:
Also, (nit) IMO moving the command to a newline with `|` isn't necessary and just clutters things a bit.

Reply:
Fixed in [c4c32f6](https://github.com/PrairieLearn/PrairieLearn/pull/11820/commits/c4c32f6776472ad1c0dd25de393b873bdeac245e).